### PR TITLE
add build discader to avoid disk full issues in jenkins

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
add a log rotator in the pipeline to avoid disk full issues

### CODE REVIEW

File: build/Jenkinsfile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env groovy
 
 pipeline {
+
+    options { buildDiscarder(logRotator(numToKeepStr: '2')) }
```
Comment (lieut-data) [inline build/Jenkinsfile]: What's the impact of the value of `2`? Can we only see results from the 2 most recent Jenkins builds? What about lots of active PRs with builds in various stages?

Comment (cpanato) [reply build/Jenkinsfile]: yes, only the last two builds, usually you will see the current build and the previous one. But I will increase this to 5.

Comment (lieut-data) [reply build/Jenkinsfile]: And just to confirm, is this `5` per PR, or `5` overall across all active PRs?

Comment (cpanato) [reply build/Jenkinsfile]: 5 build history per active PR

**Pull request merged**